src/main/java/uk/gov/indexer/Indexer.java

```diff
@@ -21,21 +28,29 @@ public Indexer(Configuration configuration, String register) {
 
     public synchronized void start(ScheduledExecutorService executorService) {
         try {
-            ConsoleLogger.log("setting up register " + register);
+            LOGGER.info("setting up register " + register);
 
             DBI destDbi = new DBI(configuration.getProperty(register + ".destination.postgres.db.connectionString"));
             DBI sourceDbi = new DBI(configuration.getProperty(register + ".source.postgres.db.connectionString"));
 
             destinationDBUpdateDAO = destDbi.open().attach(DestinationDBUpdateDAO.class);
 
-            sourceDBQueryDAO = sourceDbi.onDemand(SourceDBQueryDAO.class);
-            executorService.scheduleAtFixedRate(
-                    new IndexerTask(register, sourceDBQueryDAO, destinationDBUpdateDAO),
-                    0,
-                    10,
-                    TimeUnit.SECONDS
-            );
+            if (configuration.getCTServerEndpointForRegister(register).isPresent()) {
```

Comment:
can be simplified as below: 

           Fetcher fetcher = configuration.getCTServerEndpointForRegister(register)
                    .map(endPoint ->
                                    (Fetcher) new CTFetcher(new CTServer(endPoint))
                    ).orElse(
                            new PGFetcher(sourceDBQueryDAO)
                    );

            executorService.scheduleAtFixedRate(
                    new IndexerTask(register, fetcher, destinationDBUpdateDAO),
                    0,
                    10,
                    TimeUnit.SECONDS);